model/stripe/model.sql

```diff
@@ -0,0 +1,71 @@
+create or replace view {{env.schema}}.stripe_invoices as (
+
+  select
+    customer,
+    date,
+    forgiven,
+    subscription as subscription_id,
+    paid,
+    total
```

Comment:
This is Chris Hexton's query based on the way they do this analysis at Vero. I'm positive that it's not going to capture 100% of all potential stripe logic and that we'll have to update in the future. Because we're not actually stripe users, it's hard for us to be opinionated here. I'd prefer to just keep what he has.